authorization-service/src/utils/lambda.ts

```diff
@@ -0,0 +1,14 @@
+import { APIGatewayProxyResult } from "aws-lambda";
+
+export const generateHttpResponse = <T>(statusCode: number, response: T): APIGatewayProxyResult => {
+
```

Comment:
Please do not put empty line here (in previous file the syntax is correct)

Reply:
done